services/device-registry/src/main/fabric8/registry-pvc.yml

```diff
@@ -4,7 +4,7 @@ spec:
   resources:
     requests:
       storage: 1Mi
-  selector:
```

Comment:
@sophokles73 I had problems deploying with these options. I didn't look more deeply into it yet, just wanted to check with you about it first.

Reply:
@dejanb what kind of problems did you have with specifying resources ?

Reply:
@dejanb: I have been using these settings for quite some time now without any problems. Can you be more specific about the trouble you run into?

Reply:
@ppatierno @sophokles73 I can't reproduce it anymore, so it was probably something in my local environment. I reverted a change.